libs/utils/src/cast_vote_records.ts

```diff
@@ -38,6 +39,16 @@ export function getCurrentSnapshot(cvr: CVR.CVR): Optional<CVR.CVRSnapshot> {
   );
 }
 
+/**
+ * Returns the original snapshot of a cast vote record, or undefined if none
+ * exists. If undefined, the cast vote record is invalid.
+ */
+export function getOriginalSnapshot(cvr: CVR.CVR): Optional<CVR.CVRSnapshot> {
+  return cvr.CVRSnapshot.find((snapshot) =>
+    snapshot['@id'].includes('-original')
```

Comment:
Why would we not use the `Type` field? It seems like the `@id` should not be semantically meaningful.

Reply:
Hm right, definitely should.